certified-connectors/AgilePoint NX/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1630 @@
+          "properties": {
+            "ID": {
+              "type": "string",
+              "description": "ID.",
+              "x-ms-visibility": "internal",
+              "title": "ID"
+            },
+            "Trigger": {
+              "type": "string",
+              "description": "Trigger.",
+              "x-ms-visibility": "internal",
+              "title": "Trigger."
+            },
+            "WebhookID": {
+              "type": "string",
+              "description": "WebhookID.",
+              "title": "Webhook ID.",
+              "x-ms-visibility": "internal"
+            },
+            "WebhookName": {
+              "type": "string",
+              "description": "Webhook Name.",
+              "title": "Webhook Name.",
+              "x-ms-visibility": "internal"
+            },
+            "Source": {
+              "type": "object",
+              "properties": {
+                "ProcInstID": {
+                  "type": "string",
+                  "description": "The unique ID of a process instance.",
+                  "title": "Process Instance ID"
+                },
+                "ProcInstName": {
+                  "type": "string",
+                  "description": "A unique name that is associated with the process instance.",
+                  "title": "Process Instance Name"
+                },
+                "ProcessInitiator": {
+                  "type": "string",
+                  "description": "The user who starts (kicks off) a process.",
+                  "title": "Process Initiator"
+                },
+                "StartedDate": {
+                  "type": "string",
+                  "description": "The date when the process instance started.",
+                  "title": "Process Started Date"
+                },
+                "DefName": {
+                  "type": "string",
+                  "description": "The display name of the process model.",
+                  "title": "Process Model Name"
+                },
+                "Version": {
+                  "type": "string",
+                  "description": "The version number of the app.",
+                  "title": "App Version"
+                },
+                "SuperProcInstID": {
+                  "type": "string",
+                  "description": "A process instance ID that acts as a parent process instance of the process instance that is intended to create.",
+                  "title": "Parent Process Instance ID"
+                },
+                "WorkObjectID": {
+                  "type": "string",
+                  "description": "An ID for an object, such as a document, that is associated with the  process instance.",
+                  "title": "Work Object ID"
+                }
+              },
+              "description": "Source"
+            }
+          }
+        },
+        "description": "Response"
+      },
+      "post": {
+        "responses": {
+          "default": {

```

Comment:
@agilepointapps Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Reply:
@sriyen-msft I have done the changes accordingly.